### PR TITLE
Refresh .env.template

### PR DESCRIPTION
**Pull request by bbengfort**
Reviewed the configurations for GDS, Trtl, BFF, Sectigo, and the Admin-UI and GDS-UI and updated the `.env.template` and `docker-compose.yaml` files accordingly. 

Blocked by #392 (contains updates from that PR as well)

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -43,8 +43,11 @@ GDS_MEMBERS_CERTS=
 GDS_MEMBERS_CERT_POOL=
 
 # GDS Database Configuration
-GDS_DATABASE_URL=leveldb:///fixtures/db
+GDS_DATABASE_URL=trtl:///fixtures/db/
```
Comment (rebeccabilbro) [inline .env.template:46]: 🎉

File: containers/docker-compose.yaml [hunk 2]
```diff
@@ -136,6 +136,12 @@ services:
       - GDS_BFF_MODE=release
       - GDS_BFF_LOG_LEVEL=debug
       - GDS_BFF_CONSOLE_LOG=true
+      - GDS_BFF_TESTNET_INSECURE=true
+      - GDS_BFF_TESTNET_ENDPOINT=gds:4436
```
Comment (rebeccabilbro) [inline containers/docker-compose.yaml:140]: not 4437?

**Review by rebeccabilbro — APPROVED** [review]
Thank you very much for keeping our documentation and developer support up-to-date @bbengfort!

File: containers/docker-compose.yaml (hunk 2 above)
Comment (bbengfort) [reply containers/docker-compose.yaml:140]: Oop, actually it's supposed to be :4433 - to match the GDS bind addr to connect on; :4436 is trtl, I guess I just got used to typing it. Thanks for catching!

**Pull request merged**